src/frontend/src/types/Types.tsx

```diff
@@ -10,11 +10,9 @@ export interface Frappe {
   extras: Array<FrappeExtra>
   price: number;
   final_price: number;
-  menu_key: number;
```

Comment:
We don't want to change anything in Types.tsx. Aka: this PR is trying to undo the changes I just made to this file